core/common/src/main/java/alluxio/security/authorization/Mode.java

```diff
@@ -109,6 +109,8 @@ public Bits not() {
   private Bits mGroupBits;
   private Bits mOtherBits;
 
+  private Mode() {} // needed for JSON serialization and deserialization
```

Comment:
NIT: is there any other classes requires similar private constructor? if yes, we should also fix them together, or add comment so we don't mistakenly delete these private constructor